lib/livebook_web/live/output.ex

```diff
@@ -346,8 +346,28 @@ defmodule LivebookWeb.Output do
     """
   end
 
-  defp render_output(%{type: :error, message: message}, %{}) do
-    render_formatted_error_message(message)
+  defp render_output(%{type: :error, message: message}, %{id: id}) do
+    assigns = %{id: id, message: message}
+    ~H"""
+    <div id={@id} class="relative group/error">
+      <div
+        id={"#{@id}-message"}
+        class="whitespace-pre-wrap break-words font-editor text-gray-500"
+        role="complementary"
+        aria-label="error"
+        phx-no-format
+      ><%= ansi_string_to_html(@message) %></div>
+      <div class="absolute right-2 top-0 z-10 invisible group-hover/error:visible">
+        <button
+          class="icon-button bg-gray-100"
+          data-el-clipcopy
+          phx-click={JS.dispatch("lb:clipcopy", to: "##{@id}-message")}
+        >
+          <.remix_icon icon="clipboard-line" class="text-lg" />
+        </button>
+      </div>
+    </div>
+    """
```

Comment:
Sweet, let's move this to `render_formatted_error_message`, so the button appears in all cases we use it :)

Reply:
Makes sense, that's what I thought.

Reply:
done..